chains/manager.go

```diff
@@ -1124,16 +1128,21 @@ func (m *manager) createSnowmanChain(
 		// P-chain.
 		ctx.ValidatorState = valState
 
-		// Initialize the validator state for future chains.
-		m.validatorState = validators.NewLockedState(&ctx.Lock, valState)
+		// Initialize validatorState for future chains.
+		valState = validators.NewLockedState(&ctx.Lock, valState)
 		if m.TracingEnabled {
-			m.validatorState = validators.Trace(m.validatorState, "lockedState", m.Tracer)
```

Comment:
While not explicitly needed in this PR, I felt like using `valState` until we assigned to `m.validatorState` at the end made a bit more sense.